common/static/common/js/spec/discussion/view/discussion_thread_list_view_spec.js

```diff
@@ -83,10 +83,11 @@
         renderSingleThreadWithProps = function(props) {
             return makeView(new Discussion([new Thread(DiscussionViewSpecHelper.makeThreadWithProps(props))])).render();
         };
-        makeView = function(discussion) {
+        makeView = function(discussion, noThreadBodyPreview) {
```

Comment:
I'd recommend making this parameter be ``options``, rather than a boolean. It will be easier to read the caller because it will be clear what value is being passed.